components/blocks/cards/SignUpCard/SignUpCard.vue

```diff
@@ -133,15 +103,25 @@
         </BaseButton>
       </div>
     </div>
-  </BaseCard>
+  </Card>
 </template>
 
 <script lang="ts">
 import { defineComponent, reactive, toRefs } from '@nuxtjs/composition-api';
-import HideShowPassword from '../HideShowPassword/HideShowPassword.vue';
+import HideShowPassword from '../../../elements/buttons/HideShowPassword/HideShowPassword.vue';
```

Comment:
Avoid relative path as mentioned above

Reply:
Did you see any specific classes that need to be better? Happy to change anything if people have ideas.

Reply:
I only checked one snapshot which had classes labeled as `button button` instead of say `sign-up button` (specific class for sign-up button). Would need to check all of them to see what to choose I think